SPEC.md

```diff
@@ -123,8 +144,8 @@ NOTE: <ANY NOTES ABOUT THE PREREQUISITES OR ALTERNATE THINGS TO CONSIDER TO GET
 -	Samples should use published packages, available on NuGet or npmjs
 -	Each sample sits in its own folder
 -	Each sample folder is named as “\<\#\>. \<KEY SCENARIO INTRODUCED BY THE SAMPLE\>”
--	Each solution/ project is named as “\<KEY SCENARIO INTRODUCED BY THE SAMPLE\>”
--	C# - each sample has its own solution file
+-	Each project is named as “\<KEY SCENARIO INTRODUCED BY THE SAMPLE\>”
+
 
 ## Static Code Analysis
 
```

Comment:
I'd recommend that we add a new section for Deploy to azure. This can talk about two workflows - 
## Using CLI tools
- MSBot and AZ CLI
## Using VS publish workflow (VS only)
- Using the Azure integration in VS to publish directly from within VS.